tests/marker_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests_test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubevirt/bridge-marker/tests"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("bridge-marker", func() {
+	Describe("bridge resource reporting", func() {
+		It("should be reported only when available on node", func() {
+
+			out, err := tests.RunOnNode("node01", "sudo ip link add br-test type bridge")
+			if err != nil {
+				panic(fmt.Errorf("%v: %s", err, out))
+			}
+
+			defer tests.RunOnNode("node01", "sudo ip link del br-test")
+
+			Eventually(func() bool {
+				node, err := clientset.CoreV1().Nodes().Get("node01", v1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				capacity, reported := node.Status.Capacity["bridge.network.kubevirt.io/br-test"]
```

Comment:
nit: you should probably use the constant from another module with the prefix instead of duplicating it here.

Reply:
Import marker and use resourceNamespace?
I would have to rework the marker init for this :(

Reply:
OK if it's too much of a hassle then ignore it.